compiler/rustc_apfloat/src/lib.rs

```diff
@@ -35,7 +35,6 @@
 #![forbid(unsafe_code)]
 #![feature(iter_zip)]
 #![feature(nll)]
-#![cfg_attr(bootstrap, feature(or_patterns))]
```

Comment:
Heh so that's how I got pinged. At least the ping works. (And this change is obviously fine)